WCFDataService/Client/System/Data/Services/Client/Binding/DataServiceCollectionOfT.cs

```diff
@@ -758,15 +758,29 @@ private void InternalLoadCollection(IEnumerable<T> items)
             Debug.Assert(!(items is DataServiceQuery), "SL Client using DSQ as items...should have been caught by ValidateIteratorParameter.");
 #endif
 
+            if (this.IsTracking && this.observer != null)
```

Comment:
Can you help point out that when this path can be hit? Since for auto-generated proxy, seems the items will always to null to DataServiceCollection constructor, then Load() will not be called.